vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -537,6 +537,12 @@ components:
           items:
             type: integer
 
+    CpuFeatures:
+      type: object
+      properties:
+        amx:
+          type: bool
```

Comment:
```suggestion
          type: boolean
          default: false
```

Reply:
Ah should have seen that one, thanks! Fixed.